### ISSUE TITLE
Add instructions for ChromeOS setup

### ISSUE THREAD

**Issue by FellippeHeitor** [FellippeHeitor/qb64.org#1]

**Comment by superloach** [comment]
Should I just open a PR with the changes I proposed? I can include the icon and everything, if you want me to.

**Comment by FellippeHeitor** [comment]
That'd be great!

**Comment by superloach** [comment]
Cool, I'll get right on it.

**Comment by superloach** [comment]
So do you want to use the Chrome logo or that Chromium icon you found?

**Comment by FellippeHeitor** [comment]
Is either more "official"?

**Comment by superloach** [comment]
Not really, and Chromebrew doesn't have a logo as far as I know... actually, I might see if I can do something about *that* later. 😉
For now, I'll stick with the Chrome logo because it's simpler and more recognizable.
